docs/src/navigation/breadcrumbs/index.jsx

```diff
@@ -56,22 +72,27 @@ function BreadcrumbsExample() {
 
                 <MarkdownContainer>
                     <Heading anchorLink="children" variant="h2">
-                        Separator Icon Type
+                        Separator String
```

Comment:
Please update here as well to use the word Divider.

Reply:
done